src/shared/components/case-editor/case-edit-submit/case-edit-submit.component.ts

```diff
@@ -91,7 +91,11 @@ export class CaseEditSubmitComponent implements OnInit, OnDestroy {
 
   submit(): void {
     this.isSubmitting = true;
-    let caseEventData: CaseEventData = this.formValueService.sanitise(this.editForm.value) as CaseEventData;
+    let caseEventData: CaseEventData = {
```

Comment:
Should I `let` you away with this one? I'm not convinced...

Reply:
Haha 😄 - it's a fair catch!